### PR TITLE
Better wording for a few move descriptions

### PR DESCRIPTION
**Pull request by Jerick120**
Changed move descriptions for `Future Sight`, `Foresight`, `Miracle Eye` and `Odor Sleuth` to less confusing wording.

### CODE REVIEW

File: data/text/moves.ts [hunk 1]
```diff
@@ -2260,7 +2260,7 @@ export const MovesText: {[id: IDEntry]: MoveText} = {
 	},
 	foresight: {
 		name: "Foresight",
-		desc: "As long as the target remains active, its evasiveness stat stage is ignored during accuracy checks against it if it is greater than 0, and Normal- and Fighting-type attacks can hit the target if it is a Ghost type. Fails if the target is already affected, or affected by Miracle Eye or Odor Sleuth.",
+		desc: "As long as the target remains active, its Evasiveness stat stage is ignored during accuracy checks if it is greater than 0. Normal and Fighting type attacks can hit the target if it is a Ghost type. Fails if the target is already affected or affected by Miracle Eye or Odor Sleuth.",
```
Comment (DaWoblefet) [inline data/text/moves.ts:2263]: We don't capitalize evasiveness anywhere else.

You also shouldn't drop the hyphen. It should say Normal- or Fighting-type. It's included in [Smogon standards](https://www.smogon.com/forums/threads/spelling-and-grammar-standards.3712738/).

File: data/text/moves.ts [hunk 2]
```diff
@@ -2366,7 +2366,7 @@ export const MovesText: {[id: IDEntry]: MoveText} = {
 	},
 	futuresight: {
 		name: "Future Sight",
-		desc: "Deals damage two turns after this move is used. At the end of that turn, the damage is calculated at that time and dealt to the Pokemon at the position the target had when the move was used. If the user is no longer active at the time, damage is calculated based on the user's natural Special Attack stat, types, and level, with no boosts from its held item or Ability. Fails if this move or Doom Desire is already in effect for the target's position.",
+		desc: "Deals damage two turns after this move is used. Damage is calculated and dealt at the end of the second turn to the Pokemon at the original position of the target. If the user is no longer active, damage is calculated based on it's Special Attack stat, types and level ignoring any boosts from held items or ability. Fails if this move or Doom Desire is already in effect for the target's position.",
```
Comment (DaWoblefet) [inline data/text/moves.ts:2369]: If Future Sight is updated, Doom Desire should be updated to match.

You introduce "end of the second turn" in the second sentence when that's already clarified in the first sentence.

You used the wrong "its" for "its Special Attack stat". Using "natural Special Attack stat" also clarifies that it's the Sp. Atk stat used in the summary screen, and I also think it's helpful to clarify that it is in fact the user's Sp. Atk stat, not whatever Pokemon is currently on the field. It also doesn't "ignore" Abilities/items from Pokemon in the back for Future Sight, it just doesn't apply them.

**Review by DaWoblefet — COMMENTED** [review]
Beyond the more obvious stuff, I'm not sure these changes improve the descriptions all that much. I'll let Marty be the judge of that.

**Comment by Marty-D** [comment]
I think this is actually worse in all cases. For Foresight and friends, you're removing important information that it only happens for incoming attacks. For the rest, DaWoblefet suggested improvements and you seem to have intentionally ignored most of them, including the `it's` -> `its` error.

I'll put these on my list to look at some other time.

**Pull request closed without merging**